opus/application/static_media/js/browse.js

```diff
@@ -1356,14 +1359,16 @@ var o_browse = {
         return (opus.prefs[browse] === "gallery" ? ".op-gallery-view" : ".op-data-table-view");
     },
 
-    getStartObsLabel: function() {
-        return (opus.prefs.view === "cart" ? "cart_startobs" : "startobs");
+    getStartObsLabel: function(tab) {
+        tab = (tab === undefined ? opus.prefs.view : tab);
+        return (tab === "cart" ? "cart_startobs" : "startobs");
     },
 
     // Instantiate infiniteScroll
-    initInfiniteScroll: function(selector) {
-        let tab = opus.getViewTab();
-        let startObsLabel = o_browse.getStartObsLabel();
+    initInfiniteScroll: function(view, selector) {
+        let tab = `#${view}`;
+        let startObsLabel = o_browse.getStartObsLabel(view);
+        opus.getViewNamespace(view).galleryBoundingRect = o_browse.countGalleryImages();
```

Comment:
Note that countGalleryImages uses the "current" opus.prefs.view tab, while the rest of this function uses the view that was passed in. countGalleryImages needs to take a tab argument as well.

Reply:
getLimit() below throughout the path function (many instances) needs to take a tab argument

Reply:
also getViewNamespace needs to take a tab argument